app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if command.count(" ") != 2:
```

Comment:
```suggestion
    if command.count(" ") != 3:
```

Reply:
also check if command is `cp`

Reply:
Why do we check for 3 spaces and not 2? In the command "cp file1 file2" - 2 spaces?

Reply:
> 
Why do we check for 3 spaces and not 2? In the command "cp file1 file2" - 2 spaces?

Reply:
sorry, better use len() here isnstead of count
if len(command.split()) != 3:

Reply:
Okay, I got it. Thank you.